### DISCUSSION TITLE
Create issues on website

### DISCUSSION THREAD

**Discussion by Yashsharma1911** [GDSCITM/GDSCITM-official-website#88] [General]
Hey everyone 👋 we are in need of some new issues especially `good-first-issues` for newcomers who may expect to come after the 31st of march because of an event, this task should be handled by previous contributors because of there understanding of project let me know what you guys think

//
@kamalchakma1 
@ArunPatel02 
@Soumya-Kushwaha 
@mukadas026 
@PalakJain9 
@sudhanshutech 
@martinyis

**Comment by PalakJain9** [comment]
Hey @Yashsharma1911, here's an issue we can create for `events` tab : on clicking the `All` tab, data of past events has been displayed. But if we display the data of all events, your **website UI breaks**. Please check more about it here: https://github.com/GDSCITM/GDSCITM-official-website/pull/68 
So, if you feel that this could be a possible good first issue, I'll create it as an issue. Thanks!

**Reply by Yashsharma1911** [reply]
Thankyou @PalakJain9 yes it is an issue

**Comment by kamalchakma1** [comment]
Hi @Yashsharma1911 can we create an issue to add feedback section as per figma design?
![feedback](https://user-images.githubusercontent.com/53227040/229267243-eb149a44-1e1e-42a6-b8ea-5b6d8b0ffb6f.PNG)

**Reply by Yashsharma1911** [reply]
let's get you a new issue @kamalchakma1  to create currently these event are hard coded which is not best practice before taking these events from the database(database feature will come later) we are in need to create a separate JSON file in which all events are defined and in the component file, we can iterate that file JSON to show all events

![Screenshot 2023-04-01 at 11 13 00 AM](https://user-images.githubusercontent.com/71271069/229267787-034861ac-1169-4cf0-91e6-40f1310e351c.png)

**Reply by kamalchakma1** [reply]
@Yashsharma1911 ok then let me create an issue

**Comment by ArunPatel02** [comment]
> Hi @Yashsharma1911 can we create an issue to add feedback section as per figma design?
> ![feedback](https://user-images.githubusercontent.com/53227040/229267243-eb149a44-1e1e-42a6-b8ea-5b6d8b0ffb6f.PNG)
> 

Hi @kamalchakma1 we already have an issue related to feedback section. you can look issue #18

**Comment by Yashsharma1911** [comment]
@martinyis  @kamalchakma1 @PalakJain9 @mukadas026 @GDSCITM/techteam you all are doing some good work here thankyou for that our goal is to make this website with the power of open-source and encourage the community to do open-source I will recommend you all to start connecting with other contributors, start appreciating them of there work, start asking for review if they are mature enough

**Comment by Yashsharma1911** [comment]
@martinyis are you in college have you tried to become GDSC lead of your college?

**Comment by Soumya-Kushwaha** [comment]
@Yashsharma1911 The "Know More" option does nothing on homepage.

![image](https://user-images.githubusercontent.com/87495134/229472761-899fcbd6-0d71-4cd2-87bd-4045239c8ae8.png)

Maybe we can add this issue and scroll the page or add some other functionality?

**Reply by Yashsharma1911** [reply]
I think @Soumya-Kushwaha there is an open issue for this if not please raise an issue for this, again thankyou 😉

**Reply by Soumya-Kushwaha** [reply]
Yes there is, I just checked.

**Reply by Yashsharma1911** [reply]
@Soumya-Kushwaha does your college have GDSC or have you applied for GDSC lead?

**Comment by Soumya-Kushwaha** [comment]
Add Github profiles along with LinkedIn and twitter profiles of the leadership team also?

**Reply by Yashsharma1911** [reply]
good idea thankyou can you please raise an issue for that it will be helpful

**Reply by Soumya-Kushwaha** [reply]
> good idea thankyou can you please raise an issue for that it will be helpful

Done!

**Reply by Yashsharma1911** [reply]
👏 @Soumya-Kushwaha do you have access to assign someone? just confirming

**Reply by Soumya-Kushwaha** [reply]
Yes I have.

**Comment by anshitmishra** [comment]
which font type we are going to use in this project I would suggest Ubuntu if you have any other type of font please let me know @Yashsharma1911  @ArunPatel02

**Reply by Yashsharma1911** [reply]
we are using open sans but ubuntu seems good font to me

**Comment by anshitmishra** [comment]
![Screenshot (21)](https://user-images.githubusercontent.com/91340109/229910268-0ffd12bd-a772-4d2a-a9a2-f54097b6362d.png)

I suggest this design for the team container because our team card now not looks great. @Yashsharma1911 @ArunPatel02 
if you like this kind of card design so I can contribute to this design. 
reference for design https://light-seven.vercel.app/ 
https://light-seven.vercel.app/team

**Reply by Yashsharma1911** [reply]
hmm 🤔 thanks for your suggestion @anshitmishra I think the current design that we have for the team section looks good as it is by design that we have in Figma although others suggestions will also be appreciated// @ArunPatel02 @Soumya-Kushwaha @kamalchakma1 @PalakJain9

**Reply by ArunPatel02** [reply]
I am also on the same page as @Yashsharma1911 our current design is looking good but thanks a lot @anshitmishra for your wonderful suggestion. let's listen to others suggestions as well @Soumya-Kushwaha  @kamalchakma1 @PalakJain9  @mukadas026

**Comment by mukadas026** [comment]
Yh I think the current design actually fits well with the whole design of the page

**Comment by kamalchakma1** [comment]
From my side also I think current one is good.

**Comment by Soumya-Kushwaha** [comment]
Current design seems good to me too!

**Comment by kamalchakma1** [comment]
Hi Guys, I have notice something I don't you guys notice or not.

In Event Section when change tab from one to another tab  existing card design is still showing along with new design.This is in mobile view. Please check from your end and If all you guys are  noticing same then we can create an issue on this.

**Reply by Yashsharma1911** [reply]
ohh, I also saw that issue yes please create issue for that

**Reply by Yashsharma1911** [reply]
try if you can attach video or screenshot for that

**Comment by mukadas026** [comment]
@kamalchakma1 can you please explain further

**Comment by kamalchakma1** [comment]
@mukadas026 sure. So In event section when you switch between tab(like From All to Upcoming or Past and from Past to Upcoming or All) previous card design is showing. Check in small screen.

**Comment by Yashsharma1911** [comment]
FYI @Soumya-Kushwaha @martinyis @kamalchakma1 @mukadas026 you can assign someone or yourself if you want

**Comment by sudhanshutech** [comment]
@Soumya-Kushwaha @martinyis @kamalchakma1 @mukadas026 @PalakJain9 and rest i guess we for small purposes/discussions we should move to discord. There we have seperate channel for gdsc website so every one can give their reviews. Discord link is in readme go and join so we could discuss there and if need we can conduct weekly meets also , what do you guys think?  // @Yashsharma1911 @ArunPatel02 @anshitmishra

**Reply by anshitmishra** [reply]
I believe the weekly meet idea is great. @sudhanshutech

**Reply by Yashsharma1911** [reply]
great lets have common time

**Reply by Yashsharma1911** [reply]
what about on Thursday at 8:30 PM IST
